backend/svirt.pm

```diff
@@ -91,6 +98,13 @@ sub can_handle {
     return;
 }
 
+sub is_shutdown {
+    my ($self) = @_;
+    my $vmname = "openQA-SUT-" . get_var('VIRSH_INSTANCE', 1);
+    my $rsp = $self->run_cmd("! virsh list --state-shutoff | grep -w $vmname");
```

Comment:
And I guess your comment there also applies here: https://github.com/os-autoinst/os-autoinst/pull/623/files#r83916516
I'd say whoever fixes this first, wins ;-)